mlflow/metrics/genai/prompts/v1.py

```diff
@@ -64,6 +84,7 @@ class EvaluationModel:
     name: str
     definition: str
     grading_prompt: str
+    include_input: bool = True
```

Comment:
Do we need to change this here as well? If we just don't pass in input to the evaluation_model it doesn't render it. We can do it similar to `grading_context_columns`. Basically we can try and avoid adding more APIs.